Makefile

```diff
@@ -4,20 +4,31 @@ default: help
 .PHONY: help
 help:
 	@echo "make help              Show this help message"
-	@echo "make dev               Run the pattern library app in a development server"
 	@echo "make build             Build the package"
-	@echo "make lint              Run the code linter(s) and print any warnings"
 	@echo "make checkformatting   Check code formatting"
+	@echo "make clean             Delete development artefacts (cached files etc.)"
+	@echo "make dev               Run the pattern library app in a development server"
+	@echo "make docker            Build Docker image for the website"
 	@echo "make format            Automatically format code"
-	@echo "make test              Run the unit tests once"
+	@echo "make lint              Run the code linter(s) and print any warnings"
+	@echo "make run-docker        Run a Docker container using the image created by \`make docker\`"
 	@echo "make sure              Make sure that the formatter, linter, tests, etc all pass"
-	@echo "make clean             Delete development artefacts (cached files, "
-	@echo "                       dependencies, etc)"
+	@echo "make test              Run the unit tests once"
```

Comment:
I sorted the entries after `make help` alphabetically to make it easier to figure out where to slot any new entries in.